drivers/tee/optee/core.c

```diff
@@ -567,10 +568,55 @@ static struct platform_driver optee_driver = {
 	.remove = optee_remove,
 };
 
+static void enable_cpu_perf_counters(void *data)
```

Comment:
This would be better to have in a separate file to minimize the risk of future conflicts.